python/test/t_SquaredNormal_std.py

```diff
@@ -0,0 +1,30 @@
+#! /usr/bin/env python
+
+from __future__ import print_function
+from openturns import *
+
+TESTPREAMBLE()
+RandomGenerator.SetSeed(0)
+
```

Comment:
same for the python test, you can mimic t_arcsine_std.py

Reply:
done